js/client/modules/@arangodb/testsuites/rta_makedata.js

```diff
@@ -106,8 +106,8 @@ function makeDataWrapper (options) {
         if (this.options.forceJson) {
           args['server.force-json'] = true;
         }
-        if (!this.options.verbose) {
-          args['log.level'] = 'warning';
+        if (this.options.extremeVerbosity) {
+          args['log.level'] = ['warning', 'V8=debug'];
```

Comment:
This changes the default log level for all runs, even if one does not set `extremeVerbosity`.
Is that intended?

Reply:
I guess this was a cut'n'paste part from other places. the `rta_makedata` suite isn't as well established as many others. 
But can fix it to the old behaviour.